node/service.json

```diff
@@ -18,11 +18,11 @@
       "public": true
     },
     "custom": {
-      "path": "/sitemap-custom.xml",
+      "path": "/sitemap/sitemap-custom.xml",
       "public": true
     },
     "user": {
-      "path": "/sitemap-user-routes.xml",
+      "path": "/sitemap/sitemap-user-routes.xml",
       "public": true
     },
     "products": {
```

Comment:
Don't you need to remove these routes? Accessing them (`https://marcos--storecomponents.myvtexdev.com/sitemap-category-1-hats.xml`, for instance), gives me a blank page with error 301.